src/apps/testapps/testH3Line.c

```diff
@@ -51,10 +50,12 @@ static void h3Line_assertions(H3Index start, H3Index end) {
                  "index is a neighbor of the previous index");
         if (i > 1) {
             t_assert(
-                !H3_EXPORT(h3IndexesAreNeighbors)(line[i], line[i - 3]),
+                !H3_EXPORT(h3IndexesAreNeighbors)(line[i], line[i - 2]),
```

Comment:
Is this a bugfix in the test? If so, how did it work before?

Reply:
That must be my bug - I'm guessing that the invalid array access (would be `line[-1]` on the first access) wasn't throwing an error with stack allocation, and since the assertion is a negative one, it passes on `!h3IndexesAreNeighbors(h3Index, trash)` - it would only fail if the `trash` access gave it an H3 index that *was* a neighbor.

Not sure how I made the mistake in the first place, but good catch!